core/arch/arm/plat-imx/mmdc.c

```diff
@@ -15,6 +14,17 @@
 #include <platform_config.h>
 #include <stdint.h>
 
+/* i.MX6 */
+#define MMDC_MDMISC		0x18
+#define MDMISC_DDR_TYPE_MASK	GENMASK_32(4, 3)
+#define MDMISC_DDR_TYPE_SHIFT	0x3
+
+/* i.MX7 */
+#define DDRC_MSTR		0x0
+#define MSTR_DDR3		BIT(0)
+#define MSTR_LPDDR2		BIT(2)
+#define MSTR_LPDDR3		BIT(3)
+
```

Comment:
Why move these bit definitions from header file to c file?

Reply:
because there are only use in this file.
the share definition between this c file and the rest of the c files are the IMX_DDR_TYPE_xxxx definitions